src/components/Video/Video.tsx

```diff
@@ -41,7 +41,6 @@ export const Video = ({ url, title }: VideoProps) => {
       src={videoURL}
       title={title}
       allow="accelerometer; autoplay; encrypted-media; gyroscope;"
-      srcDoc={`<style>*{padding:0;margin:0;overflow:hidden}html,body{height:100%}img,span{position:absolute;width:100%;top:0;bottom:0;margin:auto}span{height:1.5em;text-align:center;font:48px/1.5 sans-serif;color:white;text-shadow:0 0 0.5em black}</style><a href=${videoURL}?autoplay=1><img src=https://img.youtube.com/vi/${videoID}/hqdefault.jpg alt='${title}'><span>▶</span></a>`}
```

Comment:
What's the impact of removing this?

Reply:
OK - fair enough. Was that flagging an accessibility issue?

Reply:
@Jaffacakes82  yea, the srcDoc overrides the youtube content, and you have to click it to refresh the image and then you can interact with the video.
It caused two accessibility issues: 1. users unaware of content refresh, 2. no focus colour for iframe